src/common/legacy_config_opts.h

```diff
@@ -286,6 +286,7 @@ OPTION(mon_inject_sync_get_chunk_delay, OPT_DOUBLE)  // inject N second delay on
 OPTION(mon_osd_force_trim_to, OPT_INT)   // force mon to trim maps to this point, regardless of min_last_epoch_clean (dangerous)
 OPTION(mon_mds_force_trim_to, OPT_INT)   // force mon to trim mdsmaps to this point (dangerous)
 OPTION(mon_mds_skip_sanity, OPT_BOOL)  // skip safety assertions on FSMap (in case of bugs where we want to continue anyway)
+OPTION(mon_osd_snap_trim_queue_warn_on, OPT_INT)
```

Comment:
@branch-predictor please avoid adding new options to `legacy_config_opts.h`, unless you plan to backport this change.

Reply:
For this one - I do plan to backport it to jewel.

Reply:
Also FWIW until we have a concise alternative to legacy_config_opts.h (for int types) I don't have a problem with adding new ones.  (It would be nice to sort out an alternative, though.)